pages/Chris-Boe

```diff
@@ -0,0 +1,10 @@
+# Chris Boe
+## Seattle Area, WA, USA
+## GMT -8:00
+Using *Windows 10*
+
+## Fun Facts:
+  * Graduating in May from Pacific Lutheran University w/ B.S. in Computer Science and minors in Mathematics & Music
+  * Addicted to coffee
+  * Gamer
+  * *Dogs* over ~~cats~~
```

Comment:
something is missing compared to your PR comment of
https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/480#issue-205038419

please add pic and links